crates/types/src/config/bifrost.rs

```diff
@@ -237,6 +237,14 @@ pub struct ReplicatedLogletOptions {
     ///
     /// Retry policy for log server RPCs
     pub log_server_retry_policy: RetryPolicy,
+
+    /// # In-memory RecordCache memory limit
+    ///
+    /// Optional size of record cache in bytes.
+    /// If set to None, record cache will be disabled.
+    /// Defaults: 20M
+    #[cfg_attr(feature = "schemars", schemars(with = "NonZeroByteCount"))]
+    pub record_cache_memory_size: Option<NonZeroByteCount>,
```

Comment:
toml format doesn't support nulls, so we can't unset this from the configuration file. Perhaps we should allow this to be disabled by setting the size to zero instead.

Reply:
Ah okay, good point. I wasn't sure if I should use None, or 0 as the value to disable the cache.